Nautono/RTOS_framework_v1/src/sail_eeprom.c

```diff
@@ -77,14 +82,15 @@ enum status_code EEPROM_Init(void)
 	}
 
 	// Return if an error occurred while initializing the I2C driver
+	
 	switch (I2C_Init()) {
```

Comment:
I don't like the idea of initializing the I2C inside of the EEPROM, yes we should check a flag but the I2C_Init() should be but in a more obvious place.

Reply:
I think I2C_init() is being called at other places too, but I agree we should definitely have an obvious location for that. I'll make that change tomorrow and test it.